FrontPanel/FrontPanel.cpp

```diff
@@ -208,6 +208,7 @@ namespace WPEFramework
 
         void FrontPanel::Deinitialize(PluginHost::IShell* /* service */)
         {
+	    CFrontPanel::instance()->Deinitialize();
```

Comment:
why not use
```
        CFrontPanel* CFrontPanel::instance()
        {
            static CFrontPanel instance;
            return &instance;
        }
```
?
for all addEventObserver please make sure removeEventObserver is called

Reply:
@npoltorapavlo So, CFrontPanel's instance function does something very similar to that, with extra bells and whistles. I've removed the CFrontPanel deinitialize from the FrontPanel plugin's deinitialize because, as you pointed it out, CFrontPanel is used by bother FrontPanel, and warehouse.

That said, please review the deinitilize portion of this push request, as treating CFrontPanel like it's own plugin, and in Front Panel GTest when we deinitialize FrontPanel, we explicitly call the CFrontPanel deintialize, allowing use to release the memory in CFrontPanel once we are done testing. This removed the Valgrind memory leak logs we were getting.

This is still a push request of its own, rather then getting merged with the FrontPanel GTest, because it's still adding new functionality to the CFrontPanel and I felt it'd be better for logging/tracking why it was added.